docs/operate/get-started/setup.md

```diff
@@ -30,35 +24,31 @@ aliases:
   - /get-started/installation/
 ---
 
-Get started by installing the open-source software that drives your hardware and connects your device to the cloud.
-The easiest way to do this is through the Viam app, so that your machines are automatically connected to configuration and remote operation tools.
+Get started by installing [`viam-server`](/operate/reference/viam-server/), the open-source software that drives your hardware and connects your device to the cloud.
 
-{{< expand "Supported systems" >}}
+## Prerequisite: Make sure you have a supported operating system
 
-Viam can run on any computer that runs one of the following operating systems:
+`viam-server` can run on any computer that runs one of the following operating systems:
 
 - Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
 - macOS
 - Windows
```

Comment:
This is duplicated info and probably now doesn't need to be. For Linux it provides a bit more info which you can put in the LInux tab instead.

Also please ensure this is correct. I think we'd want to cover the 32-bit ARM version we support as well?

Reply:
Ok moved the info. RE 32-bit ARM, I'm talking to Abe about it and he's going to check setup instructions for it next week. I'll track here https://viam.atlassian.net/browse/DOCS-3671